cpp/oneapi/dal/data/graph_common.hpp

```diff
@@ -22,6 +22,7 @@
 */
 
 #pragma once
+#include <type_traits>
```

Comment:
is it possible to move this include somewhere on the library level? ML algorithms also use it